.gitignore

```diff
@@ -15,9 +15,6 @@
 # Logs
 *.log
 
-# Ignore netCDF files.
```

Comment:
I'm guessing this file should not be touched.

Reply:
Fixed.